resources/r/model.R

```diff
@@ -60,6 +60,13 @@ manipulate_regions_for_shapefile <- function(region_prevalence) {
   return(thirteen_level_NHS_regional_prevalence)
 }
   
+#Add rank column/variable to dataset
+
+create_rank_variable_by_region <- function(thirteen_level_NHS_regional_prevalence){
+  thirteen_level_NHS_regional_prevalence$rank <- NA
+  thirteen_level_NHS_regional_prevalence$rank[order(-thirteen_level_NHS_regional_prevalence$prevalence)] <- 1:nrow(thirteen_level_NHS_regional_prevalence)
+  return(thirteen_level_NHS_regional_prevalence)
```

Comment:
Can you add an extra line break before the 'return' line so it's consistent with the other functions?